### PR TITLE
trackbacks: emphasis on title

### PR DESCRIPTION
**Pull request by SBBCornell**
Some html and css edits to emphasize the trackback title more and the metadata less.

When spinning it up locally I cant find a paper that has a trackback. The attached are tested with these changes in-browser on the live site, but Id love to locate a test paper with a trackback to test with also before it is pushed live. Do you know of any?

Screenshot of the new visual changes for reference:
Screen Shot 2020-07-06 at 4 32 50 PM

### CODE REVIEW

**Comment by mhl10** [comment]
@SBBCornell [0808.4142](http://localhost:5000/abs/0808.4142) is the example with [trackbacks](http://localhost:5000/tb/0808.4142). You may need to run `FLASK_APP=app.py pipenv run python populate_test_database.py --drop_and_create` in case you don't see it.

Screenshot 2020-07-06 21 26 13

The bigger trackback title looks good. I wonder about the paper metadata still, which was a carryover from the old trackback page. Users who are viewing the trackback page for an individual paper are usually clicking from the /abs page for that paper, so it may not be as useful to include the paper metadata so predominantly. Maybe the metadata could be shown when hovering over the paper ID in "Trackbacks for 0808.4142". Or an expand/collapse widget.

I would also lean towards enlarging "Trackbacks for 0808.412" with `h1` (it's `h2`now).

File: Pipfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (mhl10) [inline Pipfile.lock:1]: This file is probably ok as committed, but probably not needed for these changes

**Comment by SBBCornell** [comment]
Updated screenshot with changes from latest commit. The metadata is now hidden by default and can be toggled open:
Screen Shot 2020-07-07 at 9 29 58 AM

**Comment by SBBCornell** [comment]
latest commit reduces emphasis on the title in the trackback guide block:

Screen Shot 2020-07-10 at 10 07 48 AM

**Comment by mhl10** [comment]
@SBBCornell, while I was fixing tests I made a few other minor changes:
- fix the vertical gap on the /abs page caused by the what-is-this style
Screenshot 2020-07-22 10 51 43

- add the trackback-styles class to the main div in the /tb/recent page:

Screenshot 2020-07-22 10 50 57

**Pull request merged**